image-loader/app/controllers/ImageLoaderController.scala

```diff
@@ -84,7 +84,10 @@ class ImageLoaderController(auth: Authentication,
       } recover {
         case e: UnsupportedMimeTypeException => FailureResponse.unsupportedMimeType(e.uploadRequest, config.supportedMimeTypes).as(ArgoMediaType)
         case _: ImageProcessingException => FailureResponse.notAnImage(config.supportedMimeTypes).as(ArgoMediaType)
-        case e => InternalServerError(Json.obj("error" -> e.getMessage)).as(ArgoMediaType)
+        case _: java.io.IOException => FailureResponse.badImage(config.supportedMimeTypes).as(ArgoMediaType)
```

Comment:
In terms of file input, what would be the difference between a file that fell into L85 versus L87?